data_base/SQLAlchemy.py

```diff
@@ -49,3 +41,24 @@ def is_data_model_exists(self, data_model: DB.db.Model, where_section: dict):
             exists_query = exists_query.where(data_model.__dict__[field] == value)
 
         return self.db.session.query(exists_query).scalar()
+
+    def get_entities(self, data_model, select_section: list, attributes_filters: dict):
+        query = self._create_query(data_model, select_section, attributes_filters)
+
+        return query.all()
+
+    def get_entity(self, data_model, select_section: list, attributes_filters: dict):
+        query = self._create_query(data_model, select_section, attributes_filters)
+
+        return query.first()
```

Comment:
Change to one method instead of duplicating code

Reply:
`get_entities`  returns `query.all()` and `get_entity` returns `query.first()`

Why it's a duplicatin?

Reply:
The functions both get the same parameters and execute the same line:
`query = self._create_query(data_model, select_section, attributes_filters) `